js/tabulus.js

```diff
@@ -13,7 +13,19 @@ function Tabulus() {
     ** Main Function Object
     */
     function my(sel) {
-      if(!container) setup(sel);
+        if(!container) setup(sel);
+        d3.selectAll("a[data-toggle='modal']")
+            .each(function() {
+                var self = d3.select(this)
+                  , name = self.attr("data-name")
+                ;
+                self
```

Comment:
@curran: To see what I mean about the buttons, try this:
stop jekyll server
then run jekyll build
then check _site/index.html

you'll see this:
`  <a role="button"
        class="btn btn-default btn-block"
        data-toggle="modal"
        data-name="about"
        href="../modals/-about.html"
        data-target="#-about-modal">
    About this topic
  </a>

  <a role="button"
        class="btn btn-default btn-block"
        data-toggle="modal"
        data-name="howto"
        href="../modals/-howto.html"
        data-target="#-howto-modal">
    Using this page
  </a>
`